components/dashboard/src/components/RepositoryFinder.tsx

```diff
@@ -122,6 +153,7 @@ export default function RepositoryFinder(props: RepositoryFinderProps) {
             expanded={!props.initialValue}
             onSelectionChange={props.setSelection}
             disabled={props.disabled}
+            loading={props.loading || isLoading}
             searchPlaceholder="Paste repository URL or type to find suggestions"
```

Comment:
**issue**: I have granted all permissions for the GitHub connection, but I can not find the `gitpod-io/gitpod` by search—I only see results for repositories in my personal account. Is this expected?

<img width="510" alt="Screenshot 2023-09-14 at 20 08 58" src="https://github.com/gitpod-io/gitpod/assets/120486/d9ec2f5f-d825-48c5-892b-cd87b0a2f6b7">

Reply:
I believe that's how it works today too. If there are projects setup for repos that aren't your personal account then you would see those (today we don't indicate there are projects, but that's what causes them to show up).